linode/manager
# feat: [M3-6450] - Add icon to images not cloud-init compatible

## Description 📝

**What does this PR do?**
Adds a cloud-init-incompatible icon and tooltip to designate which of a user's custom images are not compatible with cloud-init.

## Preview 📷

**Remove this section or include a screenshot or screen recording of the change**
| Light Mode  | Dark Mode  |   
|---|---|
| ![Screenshot 2023-04-10 at 7 13 50 AM](https://user-images.githubusercontent.com/114685994/230918981-5211c38a-bd48-4b80-bb35-fdfd4f0f6fc4.jpg)  | ![Screenshot 2023-04-10 at 7 14 13 AM](https://user-images.githubusercontent.com/114685994/230918962-b847eb6b-453f-445c-8e89-e0ff07414e11.jpg)  |   



## How to test 🧪

**What are the steps to reproduce the issue or verify the changes?**
- Use a dev test account that is configured for metadata.
- Turn on the MSW _or_ create/have at least one custom image marked as cloud-init compatible and one that is not marked as cloud-init compatible. (cloud-init compatible images will have `capabilites["cloud-init"]`)
- Go to http://localhost:3000/linodes/create?type=Images and open the image select dropdown.
- Verify that cloud-init compatible images do not have the tooltip icon, and cloud-init incompatible images do.
- Go to http://localhost:3000/linodes/create?type=Distributions and, with the MSW off, verify that all distros except for distros marked cloud-init compatible (currently: Ubuntu 22.10 - CloudInit Special) display the tooltip icon.
- Check both light & dark mode and selected & focus state to make sure icon displays as expected (correct color, etc.).
